src/main/java/com/minecolonies/coremod/entity/pathfinding/Pathfinding.java

```diff
@@ -92,191 +77,4 @@ public static void enqueue(@NotNull final AbstractPathJob job)
     {
         job.getResult().startJob(getExecutor());
     }
-
-    /**
-     * Render debugging information for the pathfinding system.
-     *
-     * @param frame       entity movement weight.
-     * @param matrixStack the matrix stack to apply to.
-     */
-    @OnlyIn(Dist.CLIENT)
-    public static void debugDraw(final double frame, final PoseStack matrixStack)
```

Comment:
is this just removed? or moved elsewhere

Reply:
He moved it into a separate class

Reply:
ah didnt see that one then, kk